services/spiget/spiget-downloads.service.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const { BaseSpigetService, documentation, keywords } = require('./spiget-base')
+const { metric } = require('../../lib/text-formatters')
+const { downloadCount } = require('../../lib/color-formatters')
+
+module.exports = class SpigetDownloads extends BaseSpigetService {
+  static get route() {
+    return {
+      base: 'spiget/downloads',
+      pattern: ':resourceid',
+    }
+  }
+
```

Comment:
Sorry I missed this last time but this should have the category set to `downloads` (it's currently showing up under other because of the base class category)